backend/sirius-web-spring-collaborative/src/main/java/org/eclipse/sirius/web/spring/collaborative/projects/RepresentationEventProcessorEntry.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.spring.collaborative.projects;
+
+import java.util.Objects;
+
+import org.eclipse.sirius.web.collaborative.api.services.IRepresentationEventProcessor;
+
+import reactor.core.Disposable;
+
+/**
+ * Entry used to store the representation event processor along with additional information.
+ *
+ * @author sbegaudeau
+ */
+public class RepresentationEventProcessorEntry {
```

Comment:
I have a feeling that there is a better name/suffix than "Entry" for this concept, but can't find it at the moment.

Reply:
I had the same feeling :)